packages/matchbox/src/components/Tag/styles.js

```diff
@@ -0,0 +1,150 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const tagBase = props => `
+  display: inline-flex;
+  align-items: center;
+  white-space: nowrap;
+  padding: 1px ${tokens.spacing_300};
+  padding-right: ${props.hasRemove ? '1px' : tokens.spacing_300};
+  border-radius: ${tokens.borderRadius_pill};
+  line-height: ${tokens.lineHeight_200};
+  font-size: ${tokens.fontSize_200};
+  font-weight: ${tokens.fontWeight_normal};
+`;
+
+// Includes button reset styles
+// Key events (space, enter) inherit the onClick handler through <button>
+export const closeBase = () => `
+  border: none;
+  background: none;
+  padding: 0;
+  display: inline-flex;
+  width: 1.25rem;
+  height: 1.25rem;
+  align-items: center;
+  justify-content: center;
+  margin-left: ${tokens.spacing_100};
+  border-radius: ${tokens.borderRadius_circle};
+  transition-duration: 0.15s;
+  transition-property: background;
+`;
+
+// This margin is a little funky, however, appears to be the best route to take to add
+// spacing between icons and other tag content. Ideally, if Tag instances do not have bare
+// strings inside, then the sibling combinator could be used (i.e., `> * + svg { margin-left: 2px; }`),
+// however, that's not possible as bare strings do not count as sibling elements.
+export const content = () => `
+  & > svg {
+    margin: 0 2px;
+  }
```

Comment:
Hmm, I'm not sure I follow. The content is wrapped in a `span`, but I don't think there's a way to reference a component (icon) passed through within children?

Reply:
I see what you mean. I think the only way to handle this might be with an `<Icon/>` component or something but that should probably be saved for another day if at all.